manifests/prow/components/prow_internals_2.yaml

```diff
@@ -217,7 +217,7 @@ data:
         agent: kubernetes
         spec:
           containers:
-          - image: docker.io/prombench/prombench:2.0.1
```

Comment:
why not do it as part of the whole change to github actions with the readme update etc?